code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -491,11 +491,11 @@
 
 /obj/item/storage/box/grenades/napalm
 	name = "box of napalm grenades"
-	desc = "A box containing 3 napalm grenades."
+	desc = "A box containing 5 napalm grenades."
 	icon_state = "secbox"
 	item_state = "secbox"
 	illustration = "grenade"
-	starts_with = list(/obj/item/grenade/napalm = 3)
+	starts_with = list(/obj/item/grenade/napalm = 5)
```

Comment:
The uplink description is wrong, they should be 3 grenades, not 5, as napalm is quite long-lasting

Reply:
Is 8 TC the intended cost for three grenades?

Reply:
It was, but it can be tuned too